example/src/main/scala/JsonWebService.scala

```diff
@@ -0,0 +1,28 @@
+import zhttp.http._
+import zhttp.service.Server
+import zio._
+
+/**
+ * Example to build app using JSON web service
+ */
+object JsonWebService extends App {
+
+  case class Employee(id: String, name: String, Experience: String)
+  val employees = List(Employee("1", "abc", "1yr"), Employee("2", "def", "2yrs"), Employee("3", "xyz", "3yrs"))
+
+  //get employee details if employee exists
+  def getDetails(id: String): String = {
+    val emp = employees.filter(_.id.equals(id))
+    if (emp.isEmpty) s"""Employee doesn't exist"""
+    else s"""{"id": "${emp.head.id}", "name": "${emp.head.name}", "experience": "${emp.head.Experience}"}"""
```

Comment:
I think we should use `zio-json` for this example instead of hard coding.

Reply:
@tusharmath zio-json lacks Scala3 support: https://github.com/zio/zio-json/issues/15, that's why the build is failing. Should I use something else?